guides/release/routing/defining-your-routes.md

```diff
@@ -109,17 +109,22 @@ ember generate route posts/new
 And then add the `{{outlet}}` helper to your template where you want the nested
 template to display. You can also add a page title with the current page name (using [page-title helper](../../accessibility/page-template-considerations/#toc_page-title)), this will help users with assistive technology know where they are in the website.
 
-```handlebars {data-filename=templates/posts.hbs}
-{{page-title "Posts - Site Title"}}
-<h1>Posts</h1>
-{{!-- Display posts and other content --}}
-{{outlet}}
+```handlebars {data-filename=templates/posts.gjs}
```

Comment:
Still in draft I see, but just quickly noticed this should be `gjs` instead of `handlebars` for all code blocks.